manifests/s/Sass/Dart-Sass/1.83.4/Sass.Dart-Sass.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Sass.Dart-Sass
+PackageVersion: 1.83.4
+InstallerType: zip
+NestedInstallerType: portable

```

```suggestion
NestedInstallerType: portable
ArchiveBinariesDependOnPath: true
```

According to [Sass: Install Sass](https://sass-lang.com/install/), the installed path should be added to `$env:PATH`.